.github/workflows/build.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     - name: Set Xcode version
       uses: maxim-lobanov/setup-xcode@v1
       with:
-        xcode-version: latest-stable
+        xcode-version: "16.1"
```

Comment:
[self-review] This should be latest xcode 16.2. With the latest version the CI is failing. Attaching the CI job for reference.
https://github.com/googlemaps/google-maps-ios-utils/actions/runs/12890243952/job/35939379429

Reply:
What if we make the `OS=18.1` in L51 and L57? Based on the [installed SDKs](https://github.com/actions/runner-images/blob/main/images/macos/macos-14-Readme.md#installed-sdks) and [installed simulators](https://github.com/actions/runner-images/blob/main/images/macos/macos-14-Readme.md#installed-simulators) that come with Xcode 16.2. See other simulator-releaed issues in the [Xcode 16.2 release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-16_2-release-notes).

Reply:
I did [this change](https://github.com/googlemaps/google-maps-ios-utils/pull/522/commits/b61f6c4b01ec650946b105c29f03b269cabf99bc) `platform=iOS Simulator,OS=18.1,name=iPhone 16`, the [CI is failing](https://github.com/googlemaps/google-maps-ios-utils/actions/runs/12914293968/job/36013556705?pr=522). 

<img width="1343" alt="Screenshot 2025-01-22 at 11 41 58 PM" src="https://github.com/user-attachments/assets/9bd84a45-8f68-45b1-99ba-cebd07335721" />

Reply:
@wangela should we go ahead with xcode 16.1?

Reply:
Removed the xcode setup.